src/drivers/MemoryDriver.ts

```diff
@@ -0,0 +1,57 @@
+import { IDriver } from "./IDriver";
+
+type Table = Map<string, any>;
+
+export class MemoryDriver implements IDriver {
+    public $store = new Map<string, Table>();
+
+    public async prepare(table: string): Promise<void> {
+        this.$getOrCreateTable(table);
+    }
+
+    public $getOrCreateTable(name: string) {
+        if (this.$store.has(name)) return this.$store.get(name)!;
+        this.$store.set(name, new Map());
+        return this.$store.get(name)!;
```

Comment:
This line also give the `Forbidden non-null assertion` warning.
I would suggest making a variable for the `new Map()` and so you can use it in the set and return it as well so there won't be a need to make a non-null assertion